docs/docs/experimentation-analysis/metrics.mdx

```diff
@@ -169,6 +169,10 @@ For example, if you want to look at the Average Order Value (AOV), what you're r
 1. `Orders per User` - selects the count of orders for each user
 2. `AOV` - selects total revenue per user and picks `Orders per User` as the denominator.
 
+##### Quantile Metrics
+
+[Quantile Metrics](/statistics/quantile) are available only as Fact Tables, as described [here](/app/fact-tables).
```

Comment:
Try changing the link to `app/fact-tables#quantile-metrics`, it should take you right to the anchor.

Reply:
done